rating/contact_link.py

```diff
@@ -28,6 +28,7 @@ def rate(self):
             for link in self.check_results['hyperlinks'][url]['links']:
                 if link['text'].lower() == 'kontakt':
                     urls_with_contact_link += 1
+                    break
```

Comment:
As this change is unrelated to the other one, I suggest to remove it from the PR. I also changed that rater recently in a similar fashion.